### PR TITLE
PORTALS-2053: fix spacing in download modal

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

File: src/lib/containers/ModalDownload.tsx [hunk 1]
```diff
@@ -164,26 +164,16 @@ export default class ModalDownload extends React.Component<
...
+              <Button variant="default" onClick={this.props.onClose} style={{ marginRight: '5px' }}>
                 Cancel
-              </a>
-              <button
-                id="submitBtn"
-                style={submitBtn}
-                className="SRC-primary-background-color SRC-roundBorder SRC-whiteText"
-                type="submit"
-              >
+              </Button>
```
Comment (jay-hodgson) [inline src/lib/containers/ModalDownload.tsx:170]: Initially moved these into a Modal.Footer section, but then realized author was using the Form submit to handle the event.

File: src/lib/template_style/_form.scss [hunk 2]
```diff
@@ -19,6 +19,14 @@
   margin-top: 0px;
   margin-bottom: 0px;
 }
+// PORTALS-2053: if checkbox/radio was generated by rjsf, then we do need some space between the elements
+.rjsf {
+  .checkbox,
+  .radio {
+    margin-bottom: 10px;
```
Comment (jay-hodgson) [inline src/lib/template_style/_form.scss:26]: Ok for checkboxes and radios to generally have 0 margin top and bottom, but in the context of a rjsf generated form (like in ModalDownload) we need to add some space.

**Comment by jay-hodgson** [comment]
rjsf form example:
Screen Shot 2021-11-12 at 10 50 21 AM
non-rjsf form example:
Screen Shot 2021-11-12 at 10 50 41 AM

**Comment by jay-hodgson** [comment]
Please see https://sagebionetworks.jira.com/browse/PORTALS-2053 for a screenshot of what it looks like before the change.

**Review by nickgros — APPROVED** [review]
LGTM

**Pull request merged**